Database/getting-started-with-MySQL-DBaaS.md

```diff
@@ -0,0 +1,49 @@
+{{{
+  "title": "Getting Started with MySQL DBaaS",
+  "date": "08-14-2015",
+  "author": "Christine Parr",
+  "attachments": [],
+  "related-products" : [],
+  "contentIsHTML": false,
+  "sticky": true
```

Comment:
yup go ahead and remove the sticky line. and don't forget to remove the common on the line above so it will look like this

```
{{{
  "title": "Getting Started with MySQL DBaaS",
  "date": "08-14-2015",
  "author": "Christine Parr",
  "attachments": [],
  "related-products" : [],
  "contentIsHTML": false
}}}
```